File: Backend/DTOs/CheckInDto.cs
using Backend.Models;

namespace Backend.DTOs
{
    public class CheckInDto
    {
        public int CheckInId { get; set; }
        public int TicketId { get; set; }
        public string PassengerName { get; set; }
        public IdDocumentType IdDocumentType { get; set; }
        public string DocumentData { get; set; }
        public bool CheckInStatus { get; set; }
        public string PassengerEmail { get; set; }
    }

}

File: Frontend/src/app/app-logic/DTOs/check-in-dto.ts
import { IdDocumentType } from "../models/checkin-item";

export interface CheckInDto {
    checkInId: number;
    ticketId: number;
    passengerName: string;
    idDocumentType: IdDocumentType;
    documentData: string;
    checkInStatus: boolean;
    passengerEmail: string;
}

